modules/op-release-notes-1-9.adoc

```diff
@@ -433,4 +433,31 @@ With this update, {pipelines-title} General Availability (GA) 1.9.2 is available
 [id="fixed-issues-1-9-2_{context}"]
 == Fixed issues
 
-* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
\ No newline at end of file
+* Before this update, an OLM misconfiguration issue had been introduced in the previous version of the release, which prevented the upgrade of {product-title}. With this update, this misconfiguration issue has been fixed.
+
+[id="release-notes-1-9-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.3
+
+With this update, {pipelines-title} General Availability (GA) 1.9.3 is available on {product-title} 4.10 in addition to 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-9-3_{context}"]
+== Fixed issues
```

Comment:
Right now the `X.Y.Z` subheadings and `X.Y.Z` main headings are all `==`, as if they are peers. I recognize that this PR matches the existing content, but the team might consider making all of the `X.Y.Z` subheadings a level lower than the `X.Y.Z` main headings as a docs enhancement later. 

Meaning:
``` 
== Release notes for {pipelines-title} General Availability X.Y.Z
=== Fixed issues
=== Known issues
```